app/src/androidTest/java/com/sdp/movemeet/Activity/ActivityNotNullDescriptionTest.java

```diff
@@ -50,7 +50,8 @@
             DUMMY_DATE,
             DUMMY_DURATION,
             DUMMY_SPORT,
-            DUMMY_ADDRESS
+            DUMMY_ADDRESS,
+            DUMMY_DATE
```

Comment:
Doesn't this conflict with the DUMMY_DATE at line 50?

Reply:
Since all we're interested in is the value of the field, I don't think that it is particularly conflicting to have two times the same value. But I agree that it can lead to some confusion, so I've changed the last constant to "DUMMY_CREATION_DATE" for that matter.